github/filecoin-project.yml

```diff
@@ -2517,16 +2444,7 @@ repositories:
     collaborators:
       admin:
         - anorth
-        - jennijuju
-        - Kubuxu
-        - masih
-        - Stebalien
-      maintain:
-        - arajasek
-        - jsoares
       push:
```

Comment:
F3 engineering team should have push access: @Stebalien, @Kubuxu, @masih, even maybe the entire FilOz if there is an associated GitHub team.

Reply:
Per below (I know GitHub is hiding it), lotus-maintainers has admin so I think we're good here.

Reply:
Ah; thanks